src/openms_gui/source/VISUAL/DIALOGS/ToolsDialog.cpp

```diff
@@ -66,12 +68,15 @@ namespace OpenMS
           String ini_file,
           String default_dir,
           LayerDataBase::DataType layer_type,
-          String layer_name
-    ) :
-    QDialog(parent),
-    ini_file_(ini_file),
-    default_dir_(default_dir),
-    params_(params)
+          const String& layer_name,
+          TVToolDiscovery* tool_scanner) :
+          QDialog(parent),
```

Comment:
please indent the initializer list. Its really hard to differenciate from regular parameters.